docs/acceptance-guide.mdx

```diff
@@ -101,24 +101,24 @@ The check to see if the user has credentials that you can accept can happen full
         (optional) which issuers are acceptable
       </li>
       <li>
-        (optional) whether the credential is required
+        (optional) whether the credential is mandatory
       </li>
     </ul>
   </p>
   <p>
-    There's a match if, for each credential request for a required credential, the user has a credential of the correct type from an acceptable issuer.
+    There's a match if, for each credential request for a mandatory credential, the user has a credential of the correct type from an acceptable issuer.
```

Comment:
We might want to rewrite this section. The new `mandatory` enum isn't quite this simple.